jarvis/commands.py

```
@@ -350,6 +745,302 @@ def compact_status(self):
+                escaped = phone.replace("'", "''")
+                sql = (
+                    "SELECT "
+                    "CASE WHEN m.is_from_me=1 THEN 'You' "
+                    f"ELSE '{contact}' END, "
+                    "m.text, "
+                    "datetime(m.date/1000000000+978307200,'unixepoch','localtime') "
+                    "FROM message m "
+                    "JOIN handle h ON m.handle_id=h.ROWID "
+                    f"WHERE h.id LIKE '%{escaped}%' "
+                    "AND m.text IS NOT NULL AND m.text!='' "
+                    f"ORDER BY m.date DESC LIMIT {count}"
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)